sopt/src/main/java/dagr/sopt/cmdline/ClpAnnotation.java

```diff
@@ -1,7 +1,7 @@
 /*
  * The MIT License
  *
- * Copyright (c) 2015 Fulcrum Genomics LLC
+ * Copyright (c) $year Fulcrum Genomics
```

Comment:
I found a bunch of classes with `$year` and some of them got fixed, and I'll blame IntelliJ for breaking ones like this.